Numsense.sln

```diff
@@ -9,6 +9,16 @@ Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Numsense.UnitTests", "Numse
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Numsense.UnitTests.CSharp", "Numsense.UnitTests.CSharp\Numsense.UnitTests.CSharp.csproj", "{5FEBD8A6-3ED8-4E4F-B484-DE4CB77B6034}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{98273287-AB32-4958-B4C2-47D77A7D42F0}"
+	ProjectSection(SolutionItems) = preProject
```

Comment:
When I am working with Visual Studio I like to have all files that are edited in the solution explorer.